specification/purview/Azure.Analytics.Purview.DataMap/routes.tsp

```diff
@@ -0,0 +1,2133 @@
+  @doc("Delete an entity identified by its GUID.")
+  @route("/atlas/v2/entity/guid/{guid}")
+  @delete
+  delete is AtlasOperation<
+    {
+      @doc("The globally unique identifier of the entity.")
+      @path
+      guid: string;
+    },
+    EntityMutationResult,
+    {}
+  >;
+
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "This is Atlas API"
+  #suppress "@azure-tools/typespec-azure-core/operation-missing-api-version" "This is Atlas API, which does not require api version"
+  @doc("Get classification for a given entity represented by a GUID.")
+  @route("/atlas/v2/entity/guid/{guid}/classification/{classificationName}")
+  @get
+  getClassification is AtlasOperation<
+    {
+      @doc("The globally unique identifier of the entity.")
+      @path
+      guid: string;
+
+      @doc("The name of the classification.")
+      @path
+      classificationName: string;
+    },
+    AtlasClassification,
+    {}
+  >;
+
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "This is Atlas API"
+  #suppress "@azure-tools/typespec-azure-core/operation-missing-api-version" "This is Atlas API, which does not require api version"
+  @doc("Delete a given classification from an existing entity represented by a GUID.")
+  @route("/atlas/v2/entity/guid/{guid}/classification/{classificationName}")
+  @delete
+  removeClassification is AtlasOperation<
+    {
+      @doc("The globally unique identifier of the entity.")
+      @path
+      guid: string;
+
+      @doc("The name of the classification.")
+      @path
+      classificationName: string;
+    },
+    void,
+    {}
+  >;
+
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "This is Atlas API"
+  #suppress "@azure-tools/typespec-azure-core/operation-missing-api-version" "This is Atlas API, which does not require api version"
+  @doc("List classifications for a given entity represented by a GUID.")
+  @route("/atlas/v2/entity/guid/{guid}/classifications")
+  @get
+  getClassifications is AtlasOperation<
+    {
+      @doc("The globally unique identifier of the entity.")
+      @path
+      guid: string;
+    },
+    AtlasClassifications,
+    {}
+  >;
+
+  #suppress "@azure-tools/typespec-azure-core/request-body-problem" "This is Atlas API behavior"
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "This is Atlas API"
+  #suppress "@azure-tools/typespec-azure-core/operation-missing-api-version" "This is Atlas API, which does not require api version"
+  @doc("Add classifications to an existing entity represented by a GUID.")
+  @route("/atlas/v2/entity/guid/{guid}/classifications")
+  @post
+  addClassifications is AtlasOperation<
+    {
+      @doc("The globally unique identifier of the entity.")
+      @path
+      guid: string;
+
+      @doc("An array of classifications to be added.")
+      @body
+      body: AtlasClassification[];
```

Comment:
Looks like some of the `@body` instances are still pending an update to having the operations parameters intersect with the request body. Should be the same as the changes you added above.

Reply:
![image](https://github.com/Azure/azure-rest-api-specs/assets/78060542/0c8ba781-36fc-41b9-bd8f-da1d7f6ba372)
I have tried before but having an error here.

Reply:
Ah I see, thanks for sharing the info